### DISCUSSION TITLE
Bed Auto Leveling problem

### DISCUSSION THREAD

**Discussion by rserroyen** [jneilliii/OctoPrint-BedLevelVisualizer#409] [General]
Hello

I have an AUTO_BED_LEVELING_FEATURE Bed Auto Leveling problem.

When I do the Update Mesh Now in Bed Visualizer
G28
G29 T
the result that

but I when I try 2 move the levels with my Bed Leveling Column the result is the same

i tried with
G28
G29 T
M500

  The result is the same
![image](https://user-images.githubusercontent.com/23522994/111866137-9f2d8a80-896b-11eb-87cc-656eb0a7a229.png)

Why ?

how to register the Bed Height Topography

M500  it's not enough

Recv: Bed Height Topography:
Recv:    +--- BACK --+
Recv:    |           |
Recv:  L |    (+)    | R
Recv:  E |           | I
Recv:  F | (-) N (+) | G
Recv:  T |           | H
Recv:    |    (-)    | T
Recv:    |           |
Recv:    O-- FRONT --+
Recv:  (0,0)
Recv:  +0.06184 +0.03475 +0.14064
Recv:  -0.13763 -0.13763 -0.02189
Recv:  -0.01943 -0.02435 +0.10370
Recv: 
Recv: 
Recv: 
Recv: Bed Level Correction Matrix:
Recv: +1.000000 +0.000000 +0.000767
Recv: -0.000000 +1.000000 +0.000352
Recv: -0.000767 -0.000352 +1.000000
Recv: X:154.01 Y:181.00 Z:7.83 E:0.00 Count X: 15323 Y:17901 Z:3252
Recv: ok

Thank

**Comment by jneilliii** [comment]
Does the bed level correction matrix conne back the same energy time? That's what's getting graphed it looks like to me.
